### DISCUSSION TITLE
markdown not being rendered as HTML

### DISCUSSION THREAD

**Discussion by bsastregx** [markedjs/marked#2630] [Q&A] [answered]
Hi, 

First of all, thanks for such a great package.
I am just trying to parse markdown as HTML.

I am using this inside a web component using [stenciljs](https://stenciljs.com/).

```
   import { marked } from 'marked';
  descriptionHTML() {
    return marked.parse('# markdown-it rulezz!');
  }
```

Then, when I run the app, this is what I get:

![image](https://user-images.githubusercontent.com/54677414/198890840-491f10e8-de64-447c-8a5a-b0b74e8bddcc.png)

As you can see, it appears h1 tags are being rendered as text, not elements.

![image](https://user-images.githubusercontent.com/54677414/198890898-3abcbf24-c4cc-41ba-aa14-e0d4e6a28d08.png)

Any help will be much appreciated.

**Comment by UziTech** [comment, marked as answer]
This sounds like a problem for stenciljs. Marked just returns an HTML string it looks like stenciljs encodes it to prevent HTML security issues.

**Reply by bsastregx** [reply]
Yeap, you were right, and incredibily fast at responding. Thanks so much! 😘
